pkgs/tools/misc/rmlint/default.nix

```diff
@@ -17,15 +18,36 @@ stdenv.mkDerivation rec {
   CFLAGS="-I${stdenv.lib.getDev utillinux}/include";
 
   nativeBuildInputs = [
-    pkgconfig sphinx gettext scons
+    makeWrapper
+    pkgconfig
+    sphinx
+    gettext
+    scons
+    wrapGAppsHook
   ];
 
   buildInputs = [
-    glib json-glib libelf utillinux
+    cairo
+    glib
+    gobject-introspection
+    gtksourceview3
+    json-glib
+    libelf
+    pango
+    polkit
+    python3
+    python3.pkgs.pygobject3
+    utillinux
   ];
 
   prefixKey = "--prefix=";
 
+  postInstall = ''
+    wrapProgram $out/bin/rmlint \
```

Comment:
We usually use `wrapPython` for this.

```
nativeBuildInputs = [
  pythonPackages.wrapPython
];
```

```
  pythonPath = [ python3.pkgs.pygobject3 python3.pkgs.pycairo ];
  # not sure if this line is needed, wrapPythonProgramsIn might already prefix `$out/bin`.
  makeWrapperArgs = ["--prefix" "PATH" ":" "$out/bin"];
  postFixup = ''
    wrapPythonProgramsIn "$out/bin/rmlint" "$out $pythonPath"
  '';
```

Reply:
`wrapPythonProgramsIn` doesn't work here, as it checks for a python shebang, and `$out/bin/rmlint` itself is no python program.